CovertInfiltration/Src/CovertInfiltration/Classes/X2StrategyElement_DefaultActivities.uc

```diff
@@ -128,9 +333,7 @@ static function int GetMissionDifficultyFromMonth (XComGameState_Activity Activi
 	return Difficulty;
 }
 
-//////////////////////////////////////////////////////////
-/// Copied from X2StrategyElement_DefaultCovertActions ///
-//////////////////////////////////////////////////////////
+// Copied from X2StrategyElement_DefaultCovertActions
```

Comment:
What's the purpose of changing the comment formatting?

Reply:
Dunno, I don't remember ever consciously changing that